scripts/sensor/01-make_evoked.py

```diff
@@ -58,6 +58,10 @@ def run_evoked(cfg, subject, session=None):
             evoked.comment = evoked.comment.replace(orig_cond_name,
                                                     new_cond_name)
             all_evoked[new_cond_name] = evoked
+    elif cfg.conditions is None:
+        # Handle resting-state
+        evoked = epochs.average()
+        all_evoked['rest'] = evoked
```

Comment:
to me makes no sense to compute the evoked data for a rest task. I would just skip this script for rest

Reply:
+100